.github/workflows/codespell.yml

```diff
@@ -0,0 +1,16 @@
+# GitHub Action to automate the identification of common misspellings in text files.
+# https://github.com/codespell-project/actions-codespell
+# https://github.com/codespell-project/codespell
+name: codespell
+on: [push, pull_request]
```

Comment:
This was the case before, but it looks we are still running twice the same test on each push.
Could this simply be `on: pull_request`?

According to [this](https://frontside.com/blog/2020-05-26-github-actions-pull_request/), the workflow is triggered when a pull request is `synchronize`d, including pushes to the pull request. The `push` entry would no longer be necessary as a result.

Reply:
I suppose we could just have this run for PRs. I mainly put this here just to try it out, so we'll have to see if it's needed and if we want to have it raise an error for misspellings.